causalpy/tests/test_integration_skl_examples.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+import pytest
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.gaussian_process.kernels import ExpSineSquared, WhiteKernel
+from sklearn.linear_model import LinearRegression
+
+import causalpy as cp
+
+
+@pytest.mark.integration
+def test_did():
+    data = cp.load_data("did")
+    assert type(data) is pd.DataFrame
+    result = cp.skl_experiments.DifferenceInDifferences(
+        data,
+        formula="y ~ 1 + group + t + treated:group",
+        time_variable_name="t",
+        prediction_model=LinearRegression(),
+    )
+    assert type(result) is cp.skl_experiments.DifferenceInDifferences
+
+
+@pytest.mark.integration
+def test_rd_drinking():
+    df = (
+        cp.load_data("drinking")
+        .rename(columns={"agecell": "age"})
+        .assign(treated=lambda df_: df_.age > 21)
+        .dropna(axis=0)
+    )
+    assert type(df) is pd.DataFrame
```

Comment:
Ideally, all the `assert` statements should be at the end of the test function: Prepare all input and finally have the assert statements.

Reply:
Or even consider splitting in two tests :)

Reply:
Done 👍